pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -244,7 +244,8 @@ public static void main(String[] args) throws Exception {
         // Format BookKeeper ledger storage metadata
         if (arguments.existingBkMetadataServiceUri == null && arguments.bookieMetadataServiceUri == null) {
             ServerConfiguration bkConf = new ServerConfiguration();
-            bkConf.setMetadataServiceUri("metadata-store:" + metadataStoreUrlNoIdentifer);
+            bkConf.setMetadataServiceUri("metadata-store:"
```

Comment:
The fix seems to be simpler if we just set `bkConf.setDelimiterParsingDisabled(false);`

Reply:
Looks like it should work, with  DelimiterParsing = false, the metadataServiceUri will not be converted to list.

Is it able to add a test? Just to make sure it will not be broken again.

Reply:
It should bkConf.setDelimiterParsingDisabled(true), not false. @codelipenghui  @Jason918 .